src/Microsoft.SqlTools.ServiceLayer/DacFx/Contracts/DeploymentOptions.cs

```diff
@@ -3,13 +3,22 @@
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 //
 using System;
+using System.ComponentModel;
 using System.IO;
 using System.Linq;
 using System.Threading.Tasks;
 using Microsoft.SqlServer.Dac;
 
 namespace Microsoft.SqlTools.ServiceLayer.DacFx.Contracts
 {
+    /// <summary>
+    /// Class to define deployment option default value and the description
+    /// </summary>
+    public class DeploymentOptionProps
```

Comment:
`DeploymentOptionProp` seems like a better name since each instance is just for a single property (and consider `DeploymentOptionProperty`. Generally unless you really need to it's best to avoid shortening names to avoid possible confusion)